locales/cy/translation.json

```diff
@@ -1120,42 +1120,42 @@
+      "header": "Use the GOV.UK One Login app to prove your identity",
+      "content": {
+        "paragraph1": "The GOV.UK One Login app works by matching your face to your photo ID.",
+        "paragraph2": "First you will need to download, open and use the app. Then you will come back to this page to finish proving your identity.",
+        "subHeading1": "1. Download the GOV.UK One Login app",
+        "paragraph3": "Scan the QR code with your smartphone to download the GOV.UK One Login app.",
+        "warningText": "The app uses a face scan with flashing colours",
+        "warningTextFallback": "Warning",
         "qrCodeAlt": "QR code to download the app",
+        "subHeading2": "Help using the GOV.UK One Login app",
         "details1": {
           "summaryText": "How to scan a QR code",
-          "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur efficitur non quam eget viverra. Duis enim velit, dignissim a feugiat id, ultricies a tortor."
+          "html": "<ol class=\"govuk-list govuk-list--number\"><li>Open the camera app on your phone.</li><li>Position your camera over the QR code.</li><li>Wait until the QR code is recognised by your camera and a link appears on your screen.</li><li>Position your camera over the QR code.</li><li>Tap the link to go to the App Store (for iPhone) or Play Store (for Android).</li><li>Download the GOV.UK One Login app.</li></ol>"
         },
         "details2": {
           "summaryText": "How to download the app without a QR code",
-          "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur efficitur non quam eget viverra. Duis enim velit, dignissim a feugiat id, ultricies a tortor."
+          "html": "<ol class=\"govuk-list govuk-list--number\"><li>Keep this page open.</li><li>On your phone, open the App Store (for iPhone) or Play Store (for Android).</li><li>Search for the ‘GOV.UK One Login’ app.</li><li>Download the GOV.UK One Login app.</li></ol>"
         },
         "details3": {
           "summaryText": "I am sensitive to flashing colours",
-          "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur efficitur non quam eget viverra. Duis enim velit, dignissim a feugiat id, ultricies a tortor."
+          "html": "<p>The face scan checks that you’re a real person. It uses different colours that change less than 3 times per second.</p><p>You can <a class=\"govuk-link\" target=\"_blank\" rel=\"noopener noreferrer\" href=\"/ipv/journey/pyi-triage-mobile-download-app/anotherWay\">prove your identity another way if you think you might be sensitive to flashing colours</a>.</p>"
         },
         "details4": {
-          "summaryText": "Oes angen i mi dynnu fy sbectol?",
-          "html": "<p>Pan fyddwch yn sganio’ch wyneb, gallwch gadw’ch sbectol ymlaen cyn belled nad oes ganddynt dint.</p><p>Os ydych yn gwisgo sbectol gyda thint, bydd angen i chi eu tynnu pan fyddwch yn sganio’ch wyneb.</p>"
```

Comment:
We can actually keep this as it is (it was new content so was translated on its own in a one-off request)